Cython/Compiler/Pipeline.py

```diff
@@ -105,10 +115,10 @@ def normalize_deps(utilcodes):
         utilcode.requires = [deps.setdefault(dep, dep) for dep in utilcode.requires or ()]
 
 
-def inject_utility_code_stage_factory(context):
+def inject_utility_code_stage_factory(context, internal_c_class_entries=True):
     def inject_utility_code_stage(module_node):
         module_node.prepare_utility_code()
-        use_utility_code_definitions(context.cython_scope, module_node.scope)
+        use_utility_code_definitions(module_node.scope.cython_scope, module_node.scope)
```

Comment:
```suggestion
        module_scope = module_node.scope
        use_utility_code_definitions(module_scope.cython_scope, module_scope)
```
and use the name throughout the function.